scripts/popup.js

```diff
@@ -393,23 +393,41 @@ function show_wikibooks() {
 function noContextTip() {
   chrome.storage.sync.get(['alexa', 'domaintools', 'tweets', 'wbmsummary', 'annotations', 'tagcloud'], function (event) {
     // If none of the context is selected, grey out the button and adding tip when the user hovers
-    for (const context in event) { if (event[context]) { return $('#context-screen').click(show_all_screens) } }
-    if (!$('#ctxbox').hasClass('flip-inside')) { $('#ctxbox').addClass('flip-inside') }
+    for (const context in event) { 
+      if (event[context]) { 
+        $('#contextBtn').removeAttr('disabled')
```

Comment:
Why this is required ?

Reply:
This line is required if you initially have any context option checked in the context menu, go to an excluded URL (eg. newtab), and click on the extension icon. This line darkens the color of the 'Show Contexts' button if the user has some contexts selected and then visits on an excluded URL (Back button does the same). The dark 'Show Contexts' button tells the user that contexts are selected in the Context menu, and the flip tells the user that this is an excluded URL.

Reply:
I was not aware of, when to darkens the color of the 'Show Context' button. Anyway I got an overall idea. Thank you !

Reply:
Thank you!! :-)